http/headers/public-key-pins-report-only.json

```diff
@@ -25,32 +24,35 @@
               "version_added": false
             },
             "ie": {
-              "version_added": null
+              "version_added": false
             },
             "opera": {
-              "version_added": "33"
+              "version_added": "33",
+              "version_removed": true
             },
             "opera_android": {
-              "version_added": "33"
+              "version_added": "33",
+              "version_removed": true
             },
             "safari": {
-              "version_added": null
+              "version_added": false
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": false
             },
             "samsunginternet_android": {
               "version_added": true,
-              "version_removed": false
+              "version_removed": true
             },
             "webview_android": {
-              "version_added": true
+              "version_added": true,
```

Comment:
```suggestion
              "version_added": "46",
```

Reply:
My understanding is that WebView didn't support this. (Not a security hole. WebView works differently with regards to HTTP in ways I don't really understand.)